Settings.cs

```diff
@@ -0,0 +1,22 @@
+﻿using UnityModManagerNet;
+
+namespace DvMod.CustomFeeValidator
+{
+	public class Settings : UnityModManager.ModSettings, IDrawable
+	{
+        [Draw("Enable logging")] public bool isLoggingEnabled =
+#if DEBUG
+            true;
+#else
+            false;
+#endif
+        [Draw(Label = "Fee Validation Type", Type = DrawType.ToggleGroup, Vertical = true)] public FeeValidationType selectedValidationType = FeeValidationType.LastLoco;
```

Comment:
Let's put the annotation on the previous line to keep line length under control:

```cs
[Draw(...)]
public FeeValidationType ...
```

Reply:
✅ done